test/unit_tests/preprocessing/test_preprocess.py

```diff
@@ -117,18 +117,12 @@ def test_preprocess_windows_callable_on_object(windows_concat_ds):
                                rtol=1e-4, atol=1e-4)
 
 
-def test_scale_deprecated():
-    msg = 'Function scale is deprecated; will be removed in 0.8.0. ' \
-          'Use numpy.multiply inside a lambda function instead.'
-    with pytest.warns(FutureWarning, match=msg):
-        deprecated_scale(np.random.rand(2, 2), factor=2)
-
 
 def test_scale_continuous(base_concat_ds):
     factor = 1e6
     preprocessors = [
         Preprocessor('pick_types', eeg=True, meg=False, stim=False),
-        Preprocessor(deprecated_scale, factor=factor)
+        Preprocessor(lambda data: multiply(data, factor), factor=factor)
```

Comment:
```suggestion
        Preprocessor(multiply, factor=factor)
```
Would this also work, if yes, would be nicer, but don't know if this works @bruAristimunha ?

Reply:
I liked your suggestion @robintibor, but it didn't work

Reply:
ok pity